Makefile

```diff
@@ -677,8 +704,8 @@ $(SBOM): $(ROOTFS_TAR) $(DOCKERFILE_ADD_SCANNER)| $(INSTALLER)
 	# this all can go away, and we can read the rootfs.tar
 	# see https://github.com/anchore/syft/issues/1400
 	tar xf $< -C $(TMP_ROOTDIR) --exclude "dev/*"
-	# we need to generate the kernel sbom, because syft would give a completely different structure
-	$(DOCKERFILE_ADD_SCANNER) scan ./pkg/kernel/Dockerfile --format spdx-json > $(TMP_ROOTDIR)/kernel-sbom.spdx.json
+	# # we need to generate the kernel sbom, because syft would give a completely different structure
+	# $(DOCKERFILE_ADD_SCANNER) scan ./pkg/kernel/Dockerfile --format spdx-json > $(TMP_ROOTDIR)/kernel-sbom.spdx.json
```

Comment:
Does this mean that eve-kernel will build and publish the kernel sbom?

Reply:
yes, I have a jira ticket for it

Reply:
Presumably we need that done before this goes in master so we can continue cutting EVE releases with a complete SBOM any time we want.

Reply:
I have created a PR https://github.com/lf-edge/eve/pull/3487 to build DOCKERFILE_ADD_SCANNER tool as a container. We need to push it to docker hub and then use in Makefile.eve to generate spdx.json file. When the kernel will be unpacked this spdx.json file will be picked up automatically by SYFT